libraries/src/MVC/View/CategoryFeedView.php

```diff
@@ -11,6 +11,7 @@
 defined('JPATH_PLATFORM') or die;
 
 use Joomla\CMS\Factory;
+use Joomla\CMS\Feed\Feed;
```

Comment:
```suggestion
use Joomla\CMS\Document\Feed\FeedItem;
```
And then sort by alpha.